### DISCUSSION TITLE
how do I forward a conversation to a human attendant?

### DISCUSSION THREAD

**Discussion by LucasSaud** [vasani-arpit/WBOT#348] [Q&A]
how do I forward a conversation to a human attendant? I want to include an option in the bot so that if the user wants to, he can talk to a human attendant, how do I do that?

**Comment by vasani-arpit** [comment]
You can setup an exact number or command like "99" or "human" to trigger human handover in bot.json. add a webhook in that block which send a notification or telegram message to human agent to tell him that user XYZ needs your attention. In response block you can add something like "Someone will reach out to you shortly."
